cmd/client_login.go

```diff
@@ -0,0 +1,60 @@
+/*
+Copyright © 2022 NAME HERE <EMAIL ADDRESS>
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package cmd
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/nucleuscloud/cli/internal/utils"
+	"github.com/spf13/cobra"
+)
+
+var clientLoginCmd = &cobra.Command{
+	Use:   "client-login",
```

Comment:
I wonder...could we augment the existing `nucleus login` instead?

```
nucleus login --sa --client-id=<blah>
> Client Secret:
```
or
```
echo "SECRET" | nucleus login --sa --client-id=<blah>
```
The flag could be `--sa` or `--service-account`